orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -289,6 +289,80 @@ def test_abs_out(self):
         assert torch.equal(cpu_out_tensor, ort_out_tensor.cpu())
         assert torch.equal(ort_result.cpu(), ort_out_tensor.cpu())
 
+    def test_eq_tensor(self):
```

Comment:
refactor these to be reusable for eq and ne. Pass in the tensor or scalar input. Hoping to test 4 ops with one set in a loop.